ports/crow/portfile.cmake

```
@@ -2,9 +2,9 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO CrowCpp/crow
     REF 7e47d4c7ee548c5fd954efd82cfeed330a7823ea #0.3+2
-    SHA512 804bafe6085270470487ea304f9b1aa812f9ee29df67569fd7133ea70bc73b61dca23d6955c1fd9cf1fa7c8909d34b447437ff05844ac2ad5bb01318357dc2c0
+    SHA512 47822ccdfb259a65bde8fed0c29bb8e58bc2f29896ff186c03098eb897985307e7477aa03808a42db01df245054411f05f95c6d9ae6477d75dd94399af3bb130
```

```suggestion
    SHA512 47822ccdfb259a65bde8fed0c29bb8e58bc2f29896ff186c03098eb897985307e7477aa03808a42db01df245054411f05f95c6d9ae6477d75dd94399af3bb130
    FILE_DISAMBIGUATOR 1
```
so as to avoid colliding with people who downloaded the old version.